packages/ripple-binary-codec/src/binary.ts

```diff
@@ -93,9 +94,9 @@ function signingData(
 /**
  * Interface describing fields required for a Claim
  */
-interface ClaimObject extends JsonObject {
```

Comment:
Why remove the `extends`?

Reply:
Property 'amount' of type 'Amount' is not assignable to 'string' index type 'JSON'.

Reply:
I can add Amount to the `type JSON = string | number | boolean | null | undefined | JSON[] | JsonObject`?

Reply:
You'll need to use `AmountObject` instead of `Amount` - `Amount` is the actual binary codec type, not the JSON representation of an amount.